server/pulp/plugins/rsync/publish.py

```diff
@@ -258,6 +258,36 @@ def process_main(self):
             raise PulpCodedException(message=output)
 
 
+class UpdateLastPredistDateStep(PublishStep):
```

Comment:
docker does not have the concept of a predistributor, it has postdistributor. That said the docker has different logic that should not be impacted by this issue.